src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1059,7 +1059,16 @@ private static Symbol FindMostSpecificImplementation(Symbol interfaceMember, Nam
                                                                  ref useSiteDiagnostics,
                                                                  out var _, out var _);
```

Comment:
Are we possibly missing use-site diagnostics?

Reply:
> Are we possibly missing use-site diagnostics?

This method is not interested in any diagnostics, it doesn't report any and doesn't return any to its caller.

---
In reply to: [285291041](https://github.com/dotnet/roslyn/pull/35756#discussion_r285291041) [](ancestors = 285291041)

Reply:
How do we know that any use site diagnostics that are deposited into `useSiteDiagnostics` here would have been previously reported so that they are not missed?

---
In reply to: [285293979](https://github.com/dotnet/roslyn/pull/35756#discussion_r285293979) [](ancestors = 285293979,285291041)

Reply:
Opened [https://github.com/dotnet/roslyn/issues/35851](https://github.com/dotnet/roslyn/issues/35851) to follow up

---
In reply to: [286216141](https://github.com/dotnet/roslyn/pull/35756#discussion_r286216141) [](ancestors = 286216141,285293979,285291041)